### ISSUE TITLE
Reduce font-weight and size of home page title

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/network#365]
**Issue:** Secondary pages have light font-weight (nice!). Homepage has bold, which is too heavy. It also jumps up to 64px on wide screens, which feels too big.

<img width="1680" alt="screen shot 2017-04-27 at 1 25 07 pm" src="https://cloud.githubusercontent.com/assets/53411/25506844/af7f122a-2b76-11e7-94e9-b7da03a1853d.png">

Matthew's comment: 

> Can we just make the large heading on all pages the same? Simplify.

**Comment by taisdesouzalessa** [comment]
@xmatthewx to address your comment about font size, I did 2 studies: 

<img width="1189" alt="image" src="https://cloud.githubusercontent.com/assets/22157921/25547181/d67535b8-2c1a-11e7-9feb-4753a586c951.png">

I feel the V2 works better because there is more breathing area and the font-size makes the value proposition more prominent. The first thing users should see is the VP and making the hero shorter take that hierarchy away. Let me know what you think.

**Comment by hannahkane** [comment]
waiting on Matthew's response; marking as blocked

**Comment by xmatthewx** [comment]
This ticket is covering a few overlapping issues.

 - [ ] font weight should be light, not bold on homepage (and all headings). this is a brand bug.
 - [ ] line height should be increased so that highlight always has space between lines. this is a brand bug on the homage (and potentially all headings)
 - [ ] let's unify page hero's and headings, including homepage. we're adding complexity without any big benefit.

I'm fine with more padding/margin, though v2 looks like too much space under logo and nav. Also, when I set the heading size to 48px in my browser, it is plenty big. 56px seems large. Huge type works fine for 2-3 word phrases. Our statement is too long. And 48px has plenty of contrast with other elements on the page.

**Comment by taisdesouzalessa** [comment]
I'll use this ticket for issues number 1 and 3 (and partially number 2). I'll create another ticket for the line height of all the other pages than home.

@gvn here are the specs, following Matthew's recommendations:
<img width="1252" alt="image" src="https://cloud.githubusercontent.com/assets/22157921/25676989/0d3a1af0-2ff9-11e7-9b3c-86cc3204ffe0.png">

For tablet I think we should have the title full width:
<img width="825" alt="image" src="https://cloud.githubusercontent.com/assets/22157921/25677022/2616f868-2ff9-11e7-8d3b-a915af3f2d61.png">

NOTE: line height may change once we implement Zilla. We may have to tweak those once Zilla is in place.

**Comment by xmatthewx** [comment]
If we aren't already checking for touch devices, let's drop the 12 column width style for those devices and just do 6 column based on screen size.
